dashboard/Dockerfile

```diff
@@ -12,17 +12,19 @@ FROM node:6.11.2
 
 RUN apt-get update && \
     apt-get install -y git
-COPY package.json /dashboard/
+RUN mkdir -p /dashboard/.tmp
+RUN useradd -ms /bin/bash node-user
```

Comment:
it's a wip but before merging we should merge all RUN operations and add cleanup calls https://github.com/eclipse/che/blob/master/dashboard/Dockerfile#L15-L16

Reply:
sure :)